### PR TITLE
task #22: Success and Error Pages

### PR DESCRIPTION
**Pull request by shimaa-samir**

### CODE REVIEW

File: app/screens/Home/Result.js [hunk 1]
```diff
@@ -1,10 +1,43 @@
...
+        super(props);
+    }
+
     render() {
-        return <h1>result</h1>;
+        return <div className="container-fluid result">
+            <div className="row">
+                <div className="col-xs-12">
+                    <img className="home-logo" src="/assets/img/logo.png"/>
+                    <div className={"success result-cont " + (this.props.status == 201 ? '' : 'hidden')}>
```
Comment (yousef-samir) [inline app/screens/Home/Result.js]: please change this condition to != 200 to match all failures

File: app/screens/Home/Result.js [hunk 2]
```diff
@@ -1,10 +1,43 @@
...
+                        <img className="sign" src="/assets/img/right-mark.svg"/>
+
+                        <p className="text-center hint margin-bottom-20">
+                            Transaction added successfully!
+                        </p>
+                        <button onClick={this.props.goToHome} className="submit">
+                            Add Another
+                        </button>
+                    </div>
+                    <div className={"failed result-cont " + (this.props.status != 201 ? '' : 'hidden')}>
```
Comment (yousef-samir) [inline app/screens/Home/Result.js]: Same condition to be used here too

File: app/screens/Home/Result.js (hunk 1 above)
Comment (shimaa-samir) [inline app/screens/Home/Result.js]: the success code in this api is 201 , it doesn't return 200 on success

File: app/screens/Home/Result.js (hunk 2 above)
Comment (shimaa-samir) [inline app/screens/Home/Result.js]: same comment as previous

File: app/screens/Home/Result.js (hunk 1 above)
Comment (yousef-samir) [inline app/screens/Home/Result.js]: Then make it != 201 and replace 'hidden' with ''

**Pull request merged**